class.jetpack.php

```diff
@@ -3215,26 +3227,330 @@ function admin_connect_notice() {
 		if ( ! current_user_can( 'jetpack_connect' ) )
 			return;
 
-		$dismiss_and_deactivate_url = wp_nonce_url( Jetpack::admin_url( '?page=jetpack&jetpack-notice=dismiss' ), 'jetpack-deactivate' );
+		$dismiss_and_deactivate_url = wp_nonce_url(
+			Jetpack::admin_url( '?page=jetpack&jetpack-notice=dismiss' ),
+			'jetpack-deactivate'
+		);
 		?>
+
+		<div id="message" class="updated jp-wpcom-connect__container">
```

Comment:
This thought crossed my mine as well last night. What made me decide to do whitespace edits in this file instead of creating a new file was just the time crunch in getting this out.

I do agree that we should aggressively refactor the `Jetpack` class so that it's not a million lines long, but I'm not sure that this is the time to make that decision.